style-dictionary/color.config.js

```diff
@@ -0,0 +1,29 @@
+const StyleDictionary = require("style-dictionary");
+const getJSON = require("./utils/getJSON");
+
+const colorTokens = getJSON("./packages/uswds-tokens/colors");
+
+const USWDS_COLORS = StyleDictionary.extend({
+  source: ["packages/uswds-tokens/colors/*.json"],
+  platforms: {
+    "scss/colors": {
+      transformGroup: "scss",
+      buildPath: "packages/uswds-core/src/styles/tokens/color/",
+      files: colorTokens.map((colorToken) => ({
+        destination: `_${colorToken}.scss`,
+        mapName: `system-color-${colorToken}`,
+        format: "scss/map-deep",
+        options: {
+          themeable: false,
+        },
```

Comment:
**note:** This option prevents the system tokens from being redefined by the users. Without, each of the new color variables will have the `!default` flag.

Example:
```scss
$yellow-5: #faf3d1 !default;
$yellow-10: #f5e6af !default;
$yellow-20: #e6c74c !default;
```

Reply:
Right, we don't want our system tokens to be themeable. I was trying to showcase why we set themeable to false and what would happen without it!

Reply:
> Right, we don't want our system tokens to be themeable. I was trying to showcase why we set themeable to false and what would happen without it!

Sorry, misunderstanding on my part. I like @amyleadem's idea of adding a comment about this.

Additionally, in future code examples we should consider something to clarify our notes. For example:

```scss
// themeable: false — Current
// Users can't accidentally override system tokens.
$yellow-5: #faf3d1;
$yellow-10: #f5e6af;
$yellow-20: #e6c74c;

// themeable: true
// Users will be able override system tokens.
$yellow-5: #faf3d1 !default;
$yellow-10: #f5e6af !default;
$yellow-20: #e6c74c !default;
```

Reply:
Comment added in [3de30bf](https://github.com/uswds/uswds/pull/5898/commits/3de30bf8c6403a1c393ce8027d07d52958bd0cc1)